test/snapshot.ts

```diff
@@ -40,19 +42,19 @@ describe('Snapshot', () => {
   const PROJECT_ID = 'grape-spaceship-123';
 
   const PUBSUB = {
+    Promise:{},
```

Comment:
This is not needed anymore as the real PubSub class should be used in the test below.